src/routes/[...lang=locale]/anime/+page.svelte

```diff
@@ -329,51 +329,65 @@
 				<Chevron class="w-[1.25vw] -rotate-90" />
 			</button>
 		</div>
-		<div class="flex flex-col md:gap-[1vw] md:h-[24vw]">
+		<div class="flex flex-col md:h-[24vw] md:gap-[1vw]">
 			<span class="font-bold text-accent md:text-[1.35vw]">{$t("home.latest_episodes.title")}</span>
 			<div class="flex size-full md:gap-[0.5vw]">
 				<div
-					class="snap-y grid w-full grid-cols-2 grid-rows-auto auto-rows-min md:gap-[1.25vw] scroll-smooth overflow-y-scroll md:pr-[1.5vw] [scrollbar-color:rgba(255,255,255,0.12)transparent]"
+					class="grid-rows-auto grid w-full snap-y auto-rows-min grid-cols-2 overflow-y-scroll scroll-smooth [scrollbar-color:rgba(255,255,255,0.12)transparent] md:gap-[1.25vw] md:pr-[1.5vw]"
 					class:scrollbar-none={IS_CHROMIUM}
 				>
 					{#each latest_episodes as episode}
-						{@const has_color_palette = latest_episodes_color_palette_mapping[episode.id] !== undefined}
-						{@const color_palette = has_color_palette && rgbHex(...latest_episodes_color_palette_mapping[episode.id][0])}
+						{@const color_palette =
+							latest_episodes_color_palette_mapping[episode.id] !== undefined &&
+							rgbHex(...latest_episodes_color_palette_mapping[episode.id][0])}
+						{@const image_loaded = latest_episodes_loaded_mapping[episode.id]}
 
-						<div
-							class="snap-start w-full md:h-[5vw] bg-cover bg-center relative md:rounded-[0.75vw] border md:border-[0.15vw] border-accent/50 duration-300"
-							style="
+						{#if image_loaded}
+							<div
+								class="relative w-full snap-start border border-accent/50 bg-cover bg-center duration-300 md:h-[5vw] md:rounded-[0.75vw] md:border-[0.15vw]"
+								style="
 								background-image: url({episode.banner});
 								border-color: #{color_palette};
 							"
```

Comment:
Avoid using direct style and use `var()` with tailwind here

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/77/commits/58859ef6cf58e42a95267d9bad6f91b93a3234ee